src/redfish/ris/rmc.py

```diff
@@ -15,7 +15,8 @@
 import copy
 import shutil
 import logging
-from collections import OrderedDict, Mapping
+from collections import OrderedDict
+from collections.abc import Mapping

```

Comment:
I assume this is backwards compatible with older versions of Python?

Reply:
Looks like it works in older versions of Python.

Reply:
it should work with any python 3.x version. it does not work with python 2.x